code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -958,6 +1017,30 @@ Pass a positive integer as an argument to override a bot's default speed.
 		eject += "<BR>"
 	return eject
 
+/mob/living/simple_animal/bot/proc/insertpai(mob/user, obj/item/paicard/card)
```

Comment:
snake_case
```suggestion
/mob/living/simple_animal/bot/proc/insert_pai(mob/user, obj/item/paicard/card)
```
Also it would be nice if you could give this proc an early return format instead of multiple nested if's, so that it is more readable.

Reply:
I refactored it like you suggested, and also found a minor logic error.